index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>First mobile</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a class="my-logo" href="index.html">My Logo</a>
+      <nav>
+        <a class="bars" href="index.html"><img src="img/Union.png" alt="bars"/></a>
```

Comment:
- [ ] According to the previous reviewer you still need to fix in the navigation bar to make it look like the Figma design. you have tried and you still need to try more to make the design look like the figma design you choose  , just little touch on the header you can reduce the height to possition the nav bar and logo 
- [ ] 
![dddd](https://user-images.githubusercontent.com/92458236/235818827-c523e6a9-23e5-41ca-8aee-28b23b700516.jpg)

![ggg](https://user-images.githubusercontent.com/92458236/235819867-42b47428-a730-4966-b680-e2e7ebdb24be.jpg)